public/js/filter_spreadsheet_table.js

```diff
@@ -0,0 +1,11 @@
+jQuery(function ($) {
```

Comment:
This could be written without jQuery, which will be an advantage when we decide to remove it, as javascript by now supports  pretty much everything.